Bottom: 5c7b3bee12223d90e488d12107fbb08109b0de70
Top:    53efabc0eddf37087f88128e961a360050bc3e18
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-11 14:25:45 +0000

w32: Update cairo recipe to 1.10.2

(Probably not the latest available, but one I successfully
used with gEDA and PCB before)


---

diff --git a/w32/recipes/cairo.recipe b/w32/recipes/cairo.recipe
index 8654d5c..62cf803 100644
--- a/w32/recipes/cairo.recipe
+++ b/w32/recipes/cairo.recipe
@@ -1,8 +1,11 @@
 name=cairo
-version=1.8.8
+version=1.10.2
 release=1
 download=http://www.cairographics.org/releases
 
 case "$host" in
-  *mingw*) configure_opt=--without-x ;;
+  *mingw*)
+    configure_opt=--without-x
+    CPPFLAGS="-D ffs=__builtin_ffs"
+  ;;
 esac
